### PR TITLE
add anon and bound leases to networks editor

### PR DESCRIPTION
**Pull request by Meshiest**
#318

### CODE REVIEW

File: js/networks.js [hunk 1]
```diff
@@ -63,7 +63,11 @@ network controller
             bridge: null,
             team_mode: null,
             conduit: "10g1",
-            pbr: null
+            pbr: null,
+            allow_anon_leases: true,
```
Comment (galthaus) [inline js/networks.js]: This should default to false as well.

We could help with names- like if the name of the range is dhcp, then anon true and bound false. if host, then anon is false and bound is true.    In either case, they should default to false.  Ignore the name thing.  That should be docs.

Comment (Meshiest) [inline js/networks.js]: so show the bound or anon leases if the name is host or dhcp respectively?

Comment (Meshiest) [inline js/networks.js]: name meaning category or group?

Comment (galthaus) [inline js/networks.js]: Ignore it.  Just default them all to false.  The range parameters get these as well, I think.

**Comment by galthaus** [comment]
I've reviewed this more directly.  The four parameters go on networks.  The name comment is correct for the names.  If the name is dhcp, then anon should be true and bound should be false.  If the name is host, then bound should be true and anon should be false.  Otherwise, all should be false.  This should only be for initial defaults.  There are cases where hosts can be both anon and bound.

**Comment by zehicle** [comment]
I've got an alternate one of these pending.

**Comment by zehicle** [comment]
Resolved in #349

**Pull request closed without merging**